src/components/playback/marker-toolbar.scss

```diff
@@ -0,0 +1,169 @@
+@import "../vars";
+
+.marker-toolbar {
+  width: 114px;
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  bottom: 46px;
+  left: 50px;
+  height: $toolbar-button-height + 4px;
+  border: $toolbar-border;
+  border-radius: $toolbar-border-radius $toolbar-border-radius $toolbar-border-radius 0;
+  overflow: hidden;
+  z-index: $toolbar-z-index;
+  background-color: white;
+  &.my-work {
+    border-color: $workspace-teal-dark-1;
+  }
+  &.class-work {
+    border-color: $classwork-purple-dark-1;
+  }
```

Comment:
I believe this should be `$classwork-purple` so it matches the border of the playback area. The same is probably true for the other tabs as well.

<img width="189" alt="PlaybackLeft" src="https://user-images.githubusercontent.com/235234/180104765-5840086d-a926-4f04-b240-b773a4c0b7fb.png">

Note that there's also a white pixel row above the bottom border.